web/src/pages/Setting/setting.scss

```diff
@@ -0,0 +1,124 @@
+.setting-container{
```

Comment:
You should remove this, this directory is already pushed in current under a new name